tests/test_jx/test_deep_ops.py

```diff
@@ -820,9 +820,7 @@ def test_aggs_on_parent_and_child(self):
                     {"o": 1, "v": NULL, "s": 0},
                     {"o": 2, "v": "b", "s": 1},
                     {"o": 2, "v": "c", "s": 0},
-                    {"o": 2, "v": NULL, "s": 1},
-                    {"o": NULL, "v": "b", "s": 0},
-                    {"o": NULL, "v": "c", "s": 0}
+                    {"o": 2, "v": NULL, "s": 1}
```

Comment:
May you send me an example of "Making database to return those Null slices, makes many other tests to fail"?  I suspect the failures are because of the extra null values showing up in list and table formats: This is ok. We can change those tests to expect extra rows that represent the missing coordinates.

The `allowNulls` is a very important when set (by default) to `true`: It is responsible for aggregating  all the documents that fall outside the space defined by the edges. We can not change the tests to `allowNulls=false`.  https://github.com/mozilla/jx-sqlite/pull/74 can not be merged.